specs/boot_loader_specification.md

```diff
@@ -335,6 +335,13 @@ The following keys are recognized:
 
   Example: `architecture aa64`
 
+* `profile` refers to the profile number to use in multi-profile EFI programs.
+  See the section `multi-profile UKIs` in [the UKI Specification](unified_kernel_image.md)
+  for more information.
+  This should only be used when `uki` or `uki-url` is set to a multi-profile UKI. Behaviour is undefined for any other cases.
```

Comment:
@poettering Is "undefined" the best way to go here? "implementation-defined" might do the same and be more easier understood by readers, but I maybe we can just define a reasonable semantic here so that people always know what will happen.